freezeyt/freezer.py

```diff
@@ -62,29 +63,49 @@ async def freeze_async(app, config):
     '5xx': 'error',
 }
 
+def mime_db_mimetype(mime_db: dict, url: str) -> Optional[List[str]]:
+    """Determines file MIME type from file suffix. Decisions are made
+    by mime-db rules.
+    """
+    suffix = PurePosixPath(url).suffix
+    if suffix.startswith("."):
+        suffix = suffix[1:]
```

Comment:
[pathlib.PurePath.suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) the return value from suffix property is always (if suffix is found) with dot, so the code should be only `suffix = PurePosixPath(url).suffix[1:]` and if statement is unnecessary

Reply:
Thanks for feedback, write me your email registered in github and I will add as coauthor

Reply:
NP @nappex, I have written my mail on Slack to PM ;). Have a nice day, Jiří